CONTRIBUTING.md

```diff
@@ -169,7 +169,7 @@ to modify component to documentation then you only need to have a look at the
 `doc.js` files.
 
 These files are used to generate the documentation on the grommet website. That
-code lives at the [grommet-site](https://github.com/grommet/grommet-site repository.
+code lives at the [grommet-site](https://github.com/grommet/grommet-site repository).
```

Comment:
Nice catch but i think it should close on grommet-site not in the repository since it is not in the link.

Reply:
Thanks @oorestisime, no idea how I managed to mix it up on such a tiny change :stuck_out_tongue: